Letterbook.Core/Models/MentionVisibility.cs

```diff
@@ -6,4 +6,5 @@ public enum MentionVisibility
     Bcc,
     To,
     Cc,
+    // does this also need to include Audience and Tags?
```

Comment:
Maybe a good idea to spin this off into an issue and tag it here so it isn't lost and we can keep visibility on any discussion.

Reply:
Maybe. It's mostly an activitypub question. Or rather, it's mostly a question of how everyone else has implemented activitypub. It wouldn't make sense to me for them to reference individual actors in those fields, but it also wouldn't make sense to me for mastodon to put the `as:public` special audience in the `To` or `Cc` fields, but they do.